# This is autogenerated file to store camera and hand-eye calibration results
camera_calibration:
  # camera matrix
  K: [1362.282739878488, 0, 769.7671812887013, 0, 1361.932430070161, 548.3188293914569, 0, 0, 1]
  # distortion parameters
  D: [-0.06711757716676722, 0.1290651551572464, -0.00226813410384065, -8.407163035214223e-05, -0.08786679047237385]
hand_eye_position:
  # rotaion matrix
  rotation: [-0.00321568, -0.0135531, -0.0116504, 0.999835]
  # translation
  translation: [-0.001561951275522637, -0.08589372872499101, 0.09399786754775818]
